packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist_Scoring.res

```diff
@@ -14,3 +14,20 @@ module KLDivergence = {
       quot < 0.0 ? Error(Operation.ComplexNumberError) : Ok(-.answerElement *. logFn(quot))
     }
 }
+
+/*
+
+*/
+module LogScore = {
+  let logFn = Js.Math.log
+  let integrand = (priorElement: float, predictionElement: float, ~answer: float) => {
+    if answer == 0.0 {
+      Ok(0.0)
+    } else if predictionElement == 0.0 {
+      Ok(infinity)
+    } else {
+      let quot = predictionElement /. priorElement
+      quot < 0.0 ? Error(Operation.ComplexNumberError) : Ok(-.answer *. logFn(quot /. answer))
```

Comment:
This is effectively 
```
-.answer *. logFn(predictionElement /. priorElement /. answer)
```

Would it be possible to separate the prior part from this calculation, and optionally add it later? That way we could have separate calculations. 

We could of course convert the log division into subtraction, though I realize that might not work due to numeric edge cases.

Reply:
after revamp comment is outdated.